Formula/vile.rb

```diff
@@ -0,0 +1,31 @@
+class Vile < Formula
+  desc "Vi Like Emacs Editor"
+  homepage "https://invisible-island.net/vile/"
+  url "https://invisible-island.net/archives/vile/current/vile-9.8v.tgz"
+  sha256 "240edec7bbf3d9df48b3042754bf9854d9a233d371d50bba236ec0edd708eed5"
+  license "GPL-2.0-or-later"
+
+  depends_on "expect" => :test
```

Comment:
Or, if we need to use `expect`, we should make this `uses_from_macos`.

Reply:
There are some "tests" that can be run without expect, but they'd be limited to something like a simple version check, or using one of the included helper tools like `vile-to-html`, but I think the current test is "better".